src/screens/LoginPage/LoginPage.tsx

```diff
@@ -395,26 +400,52 @@ function LoginPage(): JSX.Element {
                         </span>
                       )}
                   </div>
-                  <label>{t('confirmPassword')}</label>
-                  <input
-                    type={show ? 'text' : 'password'}
-                    id="cpassword"
-                    placeholder={t('confirmPassword')}
-                    required
-                    value={signformState.cPassword}
-                    onChange={(e) => {
-                      setSignFormState({
-                        ...signformState,
-                        cPassword: e.target.value,
-                      });
-                    }}
-                  />
-                  <label
-                    id="showPasswordr"
-                    className={styles.showregister}
-                    onClick={handleShow}
-                    data-testid="showPasswordr"
-                  ></label>
+                  <div className={styles.passwordalert}>
+                    <label>{t('confirmPassword')}</label>
+                    <input
+                      type={showCon ? 'text' : 'password'}
+                      id="signpassword"
+                      placeholder={t('confirmPassword')}
+                      required
+                      value={signformState.cPassword}
+                      onChange={(e) => {
+                        setSignFormState({
+                          ...signformState,
+                          cPassword: e.target.value,
+                        });
+                      }}
+                      onFocus={() => setIsInputFocusedCon(true)}
+                      onBlur={() => setIsInputFocusedCon(false)}
+                      data-testid="cpassword"
+                    />
+                    <label
+                      id="showPasswordr"
+                      className={styles.showregister}
+                      onClick={handleShowCon}
+                      data-testid="showPasswordrCon"
+                    >
+                      {showCon ? (
+                        <i className="fas fa-eye"></i>
+                      ) : (
+                        <i className="fas fa-eye-slash"></i>
+                      )}
+                    </label>
+                    {isInputFocusedCon &&
+                      signformState.signPassword !==
+                        signformState.cPassword && (
+                        <span data-testid="passwordCheck">
+                          {t('Password_and_Confirm_password_mismatches.')}
+                        </span>
+                      )}
+                    {!isInputFocusedCon &&
```

Comment:
These conditions can be moved to function/s that can be re-used instead of replicating the code. As a plus it would also look cleaner within the jsx section